app/assets/stylesheets/cart.css.scss

```diff
@@ -1,30 +1,78 @@
 .cart {
-    background: white;
+  background:white;
 }
 
-.cart-picture {
-    background: transparent image-url("checkout.jpg") center;
+.cart h3 {
+  padding-top:10px;
+  text-align:center;
+  font-weight: 200;
+  font-size: 42px;
 }
 
 .cart-picture {
-    height: 375px;
+  height: 200px;
+  background: transparent image-url("checkout.jpg") center;
+  box-shadow: 0px 10px 20px rgba(0,0,0,.3);
+}
+
+.cart-container {
+  height: 445px;
+  margin-right:auto;
+  margin-left:auto
 }
 
 #cart-table {
-    margin-left:150px;
+    margin-left:50px;
+    width: 850px;
+    border: none;
+    border-radius: 5px;
+    box-shadow: 0px 10px 20px rgba(0,0,0,.3);
 }
 
 #cart-table th {
-    text-align: left;
+  text-align: left;
+  padding: 15px;
+}
+
+#cart-table td {
+  text-align: left;
+}
+
+#quantity-col {
+  width: 30px;
 }
 
+#name-col {
+  width: 100px;
+}
+#continent-col {
+  width: 100px;
+}
+#activity-col {
+  width: 100px;
+}
+#extras-col {
+  width: 400px;
+}
+#cart-table a, a:visited, a:link {
+  color: #666666;
+}
+
+#cart-total {
+    left: 350px;
+    text-align: center;
+   border: none;
+  border-radius: 5px;
+  box-shadow: 0px 10px 20px rgba(0,0,0,.3);
+}
 
-#cart-total  {
-    margin-left: 250px;
+#cart-total th, td {
+  text-align: center;
 }
 
 #checkout {
-    background: green;
+  background: rgba(0,0,0,.2);
```

Comment:
`.2` should be written with a leading zero as `0.2`<br>Commas in function arguments should be followed by a single space